api/v1/test_helpers.go

```diff
@@ -12,8 +12,8 @@ import (
 
 const (
 	YtsaurusName     = "test-ytsaurus"
-	CoreImageFirst   = "ytsaurus/ytsaurus-nightly:dev-23.1-5f8638fc66f6e59c7a06708ed508804986a6579f"
-	CoreImageSecond  = "ytsaurus/ytsaurus-nightly:dev-23.1-9779e0140ff73f5a786bd5362313ef9a74fcd0de"
+	CoreImageFirst   = "ytsaurus/ytsaurus-nightly:dev-23.1-28ccaedbf353b870bedafb6e881ecf386a0a3779"
+	CoreImageSecond  = "ytsaurus/ytsaurus-nightly:dev-23.1-28ccaedbf353b870bedafb6e881ecf386a0a3779"
```

Comment:
I see 2 problems here:
1) you have replaced two different images with same one — therefore e2e test `test-minor-update` will test different scenario which wasn't intended
2) there is a `TEST_IMAGES` list in the Makefile which (unfortunately) duplicates list of images from the current file. Images being loaded in kind before e2e test. I assume your problem with test running locally and not failing at CI is because pods are downloading images which takes more than timeout. So you must update Makefile accordingly to this file.

While we catch that problem could you please add a comment here that this list should be in sync with one in the Makefile until we figure something better.

Reply:
Fixed